pkgs/tools/wayland/swaycwd/default.nix

```
@@ -0,0 +1,45 @@
+{ pkgs, nim, stdenv, lib,
+  enableShells ? [ "bash" "zsh" "fish" "sh" "posh" ]
+}:
+stdenv.mkDerivation {
+    name = "swaycwd";
+    version = "0.0.1";
+
+    src = pkgs.fetchFromGitLab {
+      owner = "cab404";
+      repo = "swaycwd";
+      rev = "aca81695ec2102b9bca6f5bae364f69a8b9d399f";
+      hash = "sha256-MkyY3wWByQo0l0J28xKDfGtxfazVPRyZHCObl9Fszh4=";
+    };
+
+    configurePhase = ''
+        {
+        echo 'let enabledShells: seq[string] = @${builtins.toJSON enableShells}'
+        echo 'export enabledShells'
+        } | tee shells.nim
+    '';
+
+    buildInputs = [ nim ];
```

This seems to be used at build time. If it is *only* used at build time, it should be changed to:
```suggestion
    nativeBuildInputs = [ nim ];
```
otherwise, the suggestion should be inserted after the existing line instead of replacing it.